apps/prairielearn/elements/pl-file-preview/pl-file-preview.mustache

```diff
@@ -18,7 +18,7 @@
             <button class="btn btn-outline-secondary btn-sm file-preview-download">
               Download
             </button>
-            <button type="button" class="btn btn-outline-secondary btn-sm">
+            <button type="button" class="btn btn-outline-secondary btn-sm" data-toggle="collapse" data-target="#file-preview-contents-{{uuid}}-{{index}}" aria-expanded="false" aria-controls="file-preview-contents-{{uuid}}-{{index}}">
               <span class="mr-2 js-toggle-show-preview-text">Show preview</span>
               <i class="file-preview-icon fa fa-angle-down"></i>
```

Comment:
This icon is not changing when the preview is shown/hidden.

Reply:
Good catch, resolved in 6e4881f.